packages/imperative/src/config/src/ProfileInfo.ts

```diff
@@ -959,58 +823,10 @@ export class ProfileInfo {
             });
         }
 
-        if (!this.mUsingTeamConfig) {
-            // Clear out the values
-            this.mOldSchoolProfileCache = [];
-            this.mOldSchoolProfileDefaults = {};
-            // Try to get profiles and types
-            this.mOldSchoolProfileRootDir = path.join(ImperativeConfig.instance.cliHome, "profiles");
-            this.mOldSchoolProfileTypes = fs.existsSync(this.mOldSchoolProfileRootDir) ?
-                ProfileIO.getAllProfileDirectories(this.mOldSchoolProfileRootDir) : [];
-            // Iterate over the types
-            for (const profType of this.mOldSchoolProfileTypes) {
-                // Set up the profile manager and list of profile names
-                const profileManager = new CliProfileManager({ profileRootDirectory: this.mOldSchoolProfileRootDir, type: profType });
-                const profileList = profileManager.getAllProfileNames();
-                // Iterate over them all
-                for (const prof of profileList) {
-                    // Load and add to the list
-                    try {
-                        const loadedProfile = await profileManager.load({ name: prof });
-                        this.mOldSchoolProfileCache.push(loadedProfile);
-                    } catch (err) {
-                        this.mImpLogger.warn(err.message);
-                    }
-                }
-
-                try {
-                    const defaultProfile = await profileManager.load({ loadDefault: true });
-                    if (defaultProfile) { this.mOldSchoolProfileDefaults[profType] = defaultProfile.name; }
-                } catch (err) {
-                    this.mImpLogger.warn(err.message);
-                }
-            }
-        } else {
-            this.mExtendersJson = ProfileInfo.readExtendersJsonFromDisk();
-        }
-
+        this.mExtendersJson = ProfileInfo.readExtendersJsonFromDisk();
         this.loadAllSchemas();
     }
 
-    // _______________________________________________________________________
-    /**
-     * Returns an indicator of whether we are using a team configuration or
-     * old-school profiles.
-     *
-     * You must call ProfileInfo.readProfilesFromDisk() before calling this function.
-     *
-     * @returns True when we are using a team config. False means old-school profiles.
-     */
-    public get usingTeamConfig(): boolean {
-        this.ensureReadFromDisk();
-        return this.mUsingTeamConfig;
-    }
```

Comment:
There were also a few references to the `ProfileInfo.usingTeamConfig`

- https://github.com/zowe/vscode-extension-for-zowe/blob/next/packages/zowe-explorer-api/src/profiles/ProfilesCache.ts#L153-L155
- `packages/zowe-explorer/src/utils/ProfilesUtils.ts(418,26)`
- `packages/zowe-explorer/src/utils/ProfilesUtils.ts(475,26)`
- `packages/zowe-explorer/src/utils/ProfileManagement.ts(292,22)`
- `packages/zowe-explorer/src/Profiles.ts(889,22)`

Since this is vNext, it doesn't make much sense to keep this reference.
Thus I suggest using the new `ProfileInfo.onlyV1ProfilesExist` to prevent V1 operations going forward.

Reply:
Even though this requires changes for consumers, I agree that we should all completely stop using ProfileInfo.usingTeamConfig. It is confusing to someone reading the logic. Does it mean we should be trying to use a team config, or does it mean that we have actually found an existing team config? To reinforce this, the following text is part of our breaking changes document:

Removed:
- ProfileInfo.usingTeamConfig
    - To detect if a team config exists use ProfileInfo.getTeamConfig
    - To detect if only V1 profiles exist use ProfileInfo.onlyV1ProfilesExist

Finally, it is possible that some of those ZE references should be just be removed. In many cases ZE should only be trying to read a team config, so there may be no need for a test.